app/lib/rack_exceptions_app.rb

```diff
@@ -0,0 +1,18 @@
+class RackExceptionsApp
+  def self.call(env)
+    if env['REQUEST_PATH'] =~ /^\/api\/.*$/
```

Comment:
Should we use the `accept` header for this? there are other APIs too, eg `candidate-api`, `data-api`, `register-api`

Reply:
It's a good suggestion but our api-docs do not mention `Accept` or `Content-Type` request headers, so not sure we can rely on their presence. Will adapt the regular expression to match all of `/api/`, `/register-api/`, '/candidate-api/` and `/data-api/`.

Reply:
Updated